src/index.html

```diff
@@ -10,23 +10,59 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Mate academy | airbnb search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
-  </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
+    <link
+      rel="stylesheet"
+      href="big-search-bar.css"
     />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
+    <link
+      rel="stylesheet"
+      href="small-search-bar.css"
     />
+  </head>
+  <body class="page">
+    <form
+      class="form_big page__form_big"
+      action="GET"
+      data-qa="big"
+    >
+      <label
+        for="big-bar"
+        class="label form_big__label"
```

Comment:
you don't use class name of `label` anywhere in stylesheets. Remove it and fix elements inside it